app/views/posts/new.html.erb

```diff
@@ -0,0 +1,9 @@
+
+<div>
+    <h1>Add a New Post</h1>
+    <%= form_with model: @post, url: user_posts_path, method: :post, scope: :post do |form|%>
+      <%= form.text_field :title, placeholder: "Add Title"%>
+      <%= form.text_field :text, placeholder: "Add Text"%>
+      <%=form.submit%>
+    <%end%>
+</div>
```

Comment:
- [ ] Great job for adding the new.html.erb for the posts but I think you forget to add views for comments since I couldn't see any view for comments in your views folder ✅ so kindly add view for new.html.erb  which accepts input for creating comments.💯📌